ephios/core/consequences.py

```diff
@@ -43,6 +43,11 @@ def editable_consequences(user):
     return qs.filter(slug__in=map(lambda hl: hl.slug, handlers)).distinct()
 
 
+def my_pending_consequences(user):
```

Comment:
it is called "my" consequences but can be given any user. maybe just pending consequences?

Reply:
The problem is that "pending consequences" are all consequences that are pending. Here we are just talking about the pending consequences of a sepcific user tho. I think it would make the code less readable when using this method if it wss only called pending consequences. @felixrindt wanna also defend this? :D

Reply:
```suggestion
def pending_consequences_acting_on(user):
```

Reply:
Well I guess that doesn't work for a filter but well